app/src/main/res/values/strings.xml

```diff
@@ -357,6 +359,9 @@
 
     <!-- Content -->
     <string name="main_page_content">Content of main page</string>
+    <string name="main_page_content_summary">What tabs are shown on the main page</string>
+    <string name="selection">Selection</string>
+    <string name="chosenTabs">Your tabs</string>
```

Comment:
Is there a reason why you used a capital letter instead an underscore?

Reply:
Nope, fixing it right now.